frontend/src/components/explorer/gemBrowser/Metabolite.vue

```diff
@@ -184,4 +186,10 @@ export default {
 };
 </script>
 
-<style lang="scss"></style>
+<style lang="scss">
+#chebi-img {
+   max-width: 400px;
+   width:100%;
+   border: 1px solid #dbdbdb;
```

Comment:
For easier maintenance, we should try to rely on Bulma colors or [vars.scss](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/frontend/src/style/vars.scss):

```suggestion
   border: 1px solid $grey-lighter;
```